tests/lib/utils/utils-test.js

```diff
@@ -186,3 +186,38 @@ describe('parseArgs', () => {
     expect(parsedArgs).toEqual(['asd', 'qwe', 'zxc']);
   });
 });
+
+describe('getPropertyValue', () => {
```

Comment:
I think this method is so generic that we should test it using simple objects instead of parsing actual code into AST. We don't need an extra abstraction here, don't you think?